vignettes/version_policy.Rmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "R version support"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  \usepackage[utf8]{inputenc}
+  %\VignetteIndexEntry{R version support}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+R evolves every year. Each update brings many bug fixes and performance 
+improvements, but also sometimes changes the default value of some parameters
+(such as switching from `stringsAsFactors = TRUE` to `FALSE` in R 4.0.0).
+
+While there is benefit in supporting older R versions, doing so adds 
+maintenance burden, especially in a large ecosystem like **easystats** that
+contains 10 packages that are supposed to seamlessly work together. Therefore,
+**easystats** is supported (meaning each package is thoroughly tested) on the
+**current R** version, the future one (**R-devel**) and the **previous four**
+versions. This is the same as the [version policy of `tidyverse`
```

Comment:
Do we need a reference to the tidyverse to justify our decision? I'd say no.

Reply:
We are not citing them to justify our decision but rather to highlight that we are not alone in having such R version support restrictions. 

Or at least that's how I read it.

Reply:
SO, we're two out of ...? Still sounds like the gray eminence to follow.

Reply:
It's fine by me to get rid of it.